src/main.rs

```diff
@@ -35,7 +41,9 @@ fn main() {
     let api = Api::configure(token).build(&handle).unwrap();
 
     let message_lifetime = env_var("MESSAGE_LIFETIME").unwrap_or(MESSAGE_LIFETIME_DEFAULT);
-    let storage = RefCell::new(Storage::new(message_lifetime));
+    let storage_path: String =
+        env_var("STORAGE_PATH").expect("Please specify STORAGE_PATH environment variable");
```

Comment:
rustfmt did it. I tried to do my best but this is the way it works.

Reply:
`env()` returns `String` already, so enough to write `env::var(...).expect` here.